docs/installation.rst

```diff
@@ -19,28 +19,16 @@ Setting Up
 
 BookBrainz depends on having PostgreSQL, Redis, Elasticsearch and NodeJS set up and running.
 
-But first some basic configuration common to both docker and manual installation.
+But first some basic configuration common to both Docker and manual installation.
 
 Cloning
 *******
 
-To clone the repository and point the local HEAD to the latest commit in the stable branch, something like the following command should work
+To clone the repository and point the local HEAD to the latest commit in the stable branch, something like the following command should work:
 
 ::
-    
-    git clone --recurse-submodules https://github.com/metabrainz/bookbrainz-site.git
-
-Since this project makes use of git submodules, you need to use ``git clone --recurse-submodules`` to clone it.
-
-Alternatively, to manually initialize submodules, run these two commands
-::
-
-    git submodule init
-    git submodule update
 
-Currently we are using these submodules
-
-* `MonkeyDo/Lobes`_ in ``src/client/stylesheets/lobes``
+    git clone --recurse-submodules https://github.com/metabrainz/bookbrainz-site.git
```

Comment:
We don't need this flag anymore as we stopped using git submodules
```suggestion
    git clone https://github.com/metabrainz/bookbrainz-site.git
```

See https://github.com/metabrainz/bookbrainz-site/commit/9373aa0ea64f7fbeb84dfcda6b6f1188fab7ad1d